packages/lesswrong/lib/collections/tags/schema.ts

```diff
@@ -115,6 +115,8 @@ export const schema: SchemaType<DbTag> = {
     group: formGroups.advancedOptions,
     optional: true,
     ...schemaDefaultValue(0),
+    // schemaDefaultValue throws an error if this is set to null, but we want to allow that
+    onUpdate: () => {},
```

Comment:
In the long run I think it would be better to remove the `throwIfSetToNull` from `schemaDefaultValue` and just let that be handled by the optional flag, but I didn't want to do that in this PR